third_party/3/pkg_resources/__init__.pyi

```diff
@@ -202,7 +202,19 @@ class IMetadataProvider:
 
 class ResolutionError(Exception): ...
 class DistributionNotFound(ResolutionError): ...
-class VersionConflict(ResolutionError): ...
+
+class VersionConflict(ResolutionError):
+    @property
+    def dist(self) -> Any: ...
+    @property
+    def req(self) -> Any: ...
+    def report(self) -> str: ...
+    def with_context(self, required_by: Dict[str, Any]) -> ContextualVersionConflict: ...
```

Comment:
Fixed in here, again. Hope that is good to go now?

Reply:
Should just return `VersionConflict` (as common base class).